plugins/org.jkiss.dbeaver.ext.firebird/src/org/jkiss/dbeaver/ext/firebird/model/FireBirdFieldType.java

```diff
@@ -28,6 +28,8 @@
 	TIME(13, Types.TIME, DBPDataKind.DATETIME, "TIME"),
 	CHAR(14, Types.CHAR, DBPDataKind.STRING, "CHAR"),
 	BIGINT(16, Types.BIGINT, DBPDataKind.NUMERIC, "BIGINT"),
+    NUMERIC(16, Types.NUMERIC, DBPDataKind.NUMERIC, "NUMERIC"), // Equal id, but another subtype - 1
+    DECIMAL(16, Types.DECIMAL, DBPDataKind.NUMERIC, "DECIMAL"), // Equal id, but another subtype - 2
```

Comment:
Mixed tabs and spaces, you can reformat this file entirely

Reply:
I really appreciate the git story, especially string annotations. Why should I spoil it?
I diligently use the correct indentation in the new code, but not ready to format everything around.